packages/design-system/src/__generated__/figma-design-tokens.json

```diff
@@ -37,7 +37,7 @@
     },
     "lineHeights": {
       "0": {
-        "value": "16",
+        "value": "AUTO",
```

Comment:
is that something we want? @rpominov

Reply:
https://discord.com/channels/955905230107738152/1065939291479478343